assets/source/setup-guide/app/views/LandingPageApp.js

```diff
@@ -53,20 +80,7 @@ const WelcomeSection = () => {
 					</Text>
 
 					<Text variant="body">
-						<Button
-							isPrimary
-							onClick={ () =>
-								getHistory().push(
-									getNewPath(
-										{},
-										wcSettings.pinterest_for_woocommerce
-											.isSetupComplete
-											? '/pinterest/catalog'
-											: '/pinterest/onboarding'
-									)
-								)
-							}
-						>
+						<Button isPrimary onClick={ handleGetStarted }>
```

Comment:
👍 

Appreciated the bit of refactoring instead of adding it here directly,  thanks!